tests/test_channelcatalog.py

```diff
@@ -80,3 +72,11 @@ def test_XMLStationChannelCatalogCustomPath():
     yaq_inv = cat.get_inventory(DateTimeRange(), Station("CI", "YAQ"))
     assert len(yaq_inv) == 1
     assert len(yaq_inv.networks[0].stations) == 1
+
+
+class MockCatalog(ChannelCatalog):
```

Comment:
Is the code just moved? If so it's better to undo the change so you don't add to the file change history unnecessarily. And in general the fewer the changes the easier your PRs will be to review.

Reply:
@IshikaKhandelwal can you revert this change? It's not needed, right?

Reply:
It's needed. `test_channel_filter_store.py` and `test_scedc_s3store.py` in `noisepy-seis-io` imports MockCatalog.